src/mods.cpp

```diff
@@ -206,25 +220,33 @@ void ModConfiguration::addMods(const std::vector<ModSpec> &new_mods)
 void ModConfiguration::addModsFromConfig(const std::string &settings_path, const std::set<std::string> &mods)
 {
 	Settings conf;
-	std::set<std::string> load_mod_names;
+	std::map<std::string, std::string> load_mod_names;
 
 	conf.readConfigFile(settings_path.c_str());
 	std::vector<std::string> names = conf.getNames();
 	for (const std::string &name : names) {
-		if (name.compare(0,9,"load_mod_")==0 && conf.getBool(name))
-			load_mod_names.insert(name.substr(9));
+		if (name.compare(0, 9, "load_mod_") == 0) {
+			std::string modpack_path = conf.get(name);
+			if (modpack_path != "false") {
```

Comment:
How about the old `true`?

Reply:
This loop is basically reading the settings to memory. A value of "false" means to ignore the setting completely. It's in the next loop when it is checked via `is_yes` (which checks for "true" among other things) to determine whether it should be added to `addon_mods` multiple times like before, for backwards compatibility, or just from the indicated modpack path.